core/helpers.py

```diff
@@ -370,8 +371,12 @@ def get_suggested_countries_by_hs_code(sso_session_id, hs_code):
 
 
 def get_sender_ip_address(request):
-    ip, is_routable = get_client_ip(request)
-    return ip or None
+    try:
+        x_forwarded_for = request.META['HTTP_X_FORWARDED_FOR']
+        client_ip = x_forwarded_for.split(',')[-3].strip()
+        return client_ip
+    except (KeyError, IndexError, GeoIP2Exception) as e:
```

Comment:
This won't throw that error
```suggestion
    except (KeyError, IndexError) as e:
```